tests/mappers/test_exon_genomic_coords.py

```diff
@@ -599,255 +911,137 @@ async def test_tpm3(
-    params = g_to_t_resp.genomic_data
-    t_to_g_resp = await test_egc_mapper.transcript_to_genomic_coordinates(
-        params.transcript,
-        gene=params.gene,
-        exon_start=params.exon_start,
-        exon_start_offset=params.exon_start_offset,
-        exon_end=params.exon_end,
-        exon_end_offset=params.exon_end_offset,
+    g_to_t_resp = await test_egc_mapper.genomic_to_tx_segment(**inputs)
+    genomic_tx_seg_service_checks(g_to_t_resp, tpm3_exon8_g)
+
+    t_to_g_resp = await test_egc_mapper.tx_segment_to_genomic(
+        **get_t_to_g_args(g_to_t_resp)
     )
-    genomic_data_assertion_checks(t_to_g_resp, tpm3_exon8_g)
+    genomic_tx_seg_service_checks(t_to_g_resp, tpm3_exon8_g)
 
 
 @pytest.mark.asyncio()
 async def test_braf(test_egc_mapper, mane_braf):
-    """Test BRAF genomic_to_transcript_exon_coordinates and
-    transcript_to_genomic_coordinates.
+    """Test BRAF genomic_to_tx_segment and
+    tx_segment_to_genomic.
     """
     inputs = {
-        "alt_ac": "NC_000007.13",
-        "start": 140501360,
-        "end": 140453136,
-        "strand": Strand.NEGATIVE,
+        "genomic_accession": "NC_000007.13",
+        "genomic_start": 140501359,  # GRCh38 coords: 140801559
+        "genomic_end": 140453136,  # GRCh38 coords: 140753336
         "gene": "BRAF",
     }
     # MANE
-    g_to_t_resp = await test_egc_mapper.genomic_to_transcript_exon_coordinates(**inputs)
-    genomic_data_assertion_checks(g_to_t_resp, mane_braf)
+    g_to_t_resp = await test_egc_mapper.genomic_to_tx_segment(**inputs)
+    genomic_tx_seg_service_checks(g_to_t_resp, mane_braf)
 
-    # no strand
-    inputs = {
-        "alt_ac": "NC_000007.13",
-        "start": 140501360,
-        "end": 140453136,
-        "gene": "BRAF",
-    }
-    g_to_t_resp = await test_egc_mapper.genomic_to_transcript_exon_coordinates(**inputs)
-    genomic_data_assertion_checks(g_to_t_resp, mane_braf)
-
-    params = g_to_t_resp.genomic_data
-    t_to_g_resp = await test_egc_mapper.transcript_to_genomic_coordinates(
-        params.transcript,
-        gene=params.gene,
-        exon_start=params.exon_start,
-        exon_start_offset=params.exon_start_offset,
-        exon_end=params.exon_end,
-        exon_end_offset=params.exon_end_offset,
+    t_to_g_resp = await test_egc_mapper.tx_segment_to_genomic(
+        **get_t_to_g_args(g_to_t_resp)
     )
-    genomic_data_assertion_checks(t_to_g_resp, mane_braf)
+    genomic_tx_seg_service_checks(t_to_g_resp, mane_braf)
 
 
 @pytest.mark.asyncio()
 async def test_wee1(test_egc_mapper, wee1_exon2_exon11, mane_wee1_exon2_exon11):
-    """Test WEE1 genomic_to_transcript_exon_coordinates and
-    transcript_to_genomic_coordinates.
+    """Test WEE1 genomic_to_tx_segment and
+    tx_segment_to_genomic.
     """
     inputs = {
-        "alt_ac": "NC_000011.9",
-        "start": 9597640,
-        "end": 9609996,
-        "strand": Strand.POSITIVE,
+        "genomic_accession": "NC_000011.9",
```

Comment:
should we be using accession `NC_000011.10` instead?

Reply:
I don't think so. This is a good example because it demonstrates lifting over to GRCh38

Reply:
Oh that makes sense in the context of liftover